low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -47,5 +47,91 @@
+                "Power Supply Redundancy will be affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please replace the faulty Power supply soon."
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "No Failure threat anymore for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Failure detected", "Failure detected ()"], {
+            "Asserted": Alert(
+                "fault", "error",
+                "Failure detected for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please replace the faulty Power Supply immediately."
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "'Power Supply {}' recovered from the failure. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Config Error", "Config Error ()"], {
+            "Asserted": Alert(
+                "fault", "error",
+                "'Power Supply {}' has configuration error. [{}]",
+                "Power Supply Redundancy is affected. If error is not fixed, "+
+                "and other power supplies go down, server may go offline.",
+                DEFAULT_RECOMMENDATION
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "Configuration error resolved for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Power Supply AC lost", "Power Supply AC lost ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "'Power Supply {}' lost the AC supply. [{}]",
+                "Power Supply Redundancy is affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please Plug on the AC Power Supply Redundancy."
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "'Power Supply {}' regained the AC Supply. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Power Supply Inactive",
+                         "Power Supply Inactive ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "'Power Supply {}' is inactive. [{}]",
+                "Power Supply Redundancy is affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please Plug on the Power Supply Redundancy."
```

Comment:
Recommendation -> Try removing Power supply unit & inserting back, and check if recovered. If seen inactive still, PSU may need replacement.